packages/tasks/src/snippets/js.ts

```diff
@@ -141,36 +189,68 @@ console.log(chatCompletion.choices[0].message);`,
 			];
 		}
 	} else {
-		return snippetBasic(model, accessToken);
+		return snippetBasic(model, accessToken, provider);
 	}
 };
 
-export const snippetZeroShotClassification = (model: ModelDataMinimal, accessToken: string): InferenceSnippet => ({
-	content: `async function query(data) {
-	const response = await fetch(
-		"https://api-inference.huggingface.co/models/${model.id}",
+export const snippetZeroShotClassification = (model: ModelDataMinimal, accessToken: string): InferenceSnippet[] => {
+	return [
 		{
-			headers: {
-				Authorization: "Bearer ${accessToken || `{API_TOKEN}`}",
-				"Content-Type": "application/json",
-			},
-			method: "POST",
-			body: JSON.stringify(data),
+			client: "fetch",
+			content: `async function query(data) {
+			const response = await fetch(
+				"https://api-inference.huggingface.co/models/${model.id}",
+				{
+					headers: {
+						Authorization: "Bearer ${accessToken || `{API_TOKEN}`}",
+						"Content-Type": "application/json",
+					},
+					method: "POST",
+					body: JSON.stringify(data),
+				}
+			);
+			const result = await response.json();
+			return result;
 		}
-	);
-	const result = await response.json();
-	return result;
-}
+		
+		query({"inputs": ${getModelInputSnippet(
+				model
+			)}, "parameters": {"candidate_labels": ["refund", "legal", "faq"]}}).then((response) => {
+			console.log(JSON.stringify(response));
+		});`,
+		},
+	];
+};
 
-query({"inputs": ${getModelInputSnippet(
-		model
-	)}, "parameters": {"candidate_labels": ["refund", "legal", "faq"]}}).then((response) => {
-	console.log(JSON.stringify(response));
-});`,
+export const snippetTextToImage = (
+	model: ModelDataMinimal,
+	accessToken: string,
+	provider: InferenceProvider
+): InferenceSnippet[] => {
+	return [
+		{
```

Comment:
Should we keep this?
If yes, let's add install instructions to other snippets

Reply:
yes fine to remove imo